codecs/h264_packet.go

```diff
@@ -213,8 +213,6 @@ func (p *H264Packet) IsDetectedFinalPacketInSequence(rtpPacketMarketBit bool) bo
 func (p *H264Packet) Unmarshal(payload []byte) ([]byte, error) {
 	if payload == nil {
 		return nil, errNilPacket
-	} else if len(payload) <= 2 {
```

Comment:
This must be readded and changed into `if len(payload) <= 1`, otherwise the function crashes when argument is `[]byte{}`

Reply:
Thanks for taking the time for review.
Fixed.